src/calculateRank.js

```diff
@@ -36,6 +38,8 @@ function calculateRank({
     PRS_WEIGHT = 3;
   const ISSUES_MEDIAN = 25,
     ISSUES_WEIGHT = 1;
+  const REVIEWS_MEDIAN = 10,
```

Comment:
@francois-rozet do you think these are okay given that less than 10% of users have more than 4 reviews?

Reply:
Oh good catch, yes the median is too high. Maybe 1 or 2?

Reply:
@francois-rozet, thanks for the quick response! I would love to have it at `20` since I want people to do more PR reviews :smiling_imp:. But given the data we aquired, that would be unfair :rofl:. Let's do `2` 👍🏻.